docs/releases/1.10.txt

```diff
@@ -910,6 +910,9 @@ Miscellaneous
 * Private API ``django.forms.models.model_to_dict()`` returns a queryset rather
   than a list of primary keys for ``ManyToManyField``\s .
 
+* The :ttag:`static` template tag now encodes ``#`` and converts it to ``%23``.
```

Comment:
It would be nice to add an explanation of why the behavior has changed (cf546e11ac76c8dec527e39ff8ce8249a195ab42). The behavior only changes if `django.contrib.staticfiles` is installed. I have a feeling the behavior difference might be nuanced, e.g. the entire URL might be encoded.